pandas/tests/io/parser/parse_dates.py

```diff
@@ -510,3 +513,148 @@ def test_parse_date_time_multi_level_column_name(self):
         expected = DataFrame(expected_data,
                              columns=['date_time', ('A', 'a'), ('B', 'b')])
         tm.assert_frame_equal(result, expected)
+
+    def test_parse_date_time(self):
```

Comment:
things that are strictly related the the actual parsing (e.g. just involve conv) can be put in a separate class; name it TestConverters (otherwise it won't be run)